Tensile/Contractions.py

```diff
@@ -307,10 +307,8 @@ def CompoundPredicates(cls, state, problemType):
             rv += [cls('LeadingFree0SizesGreaterOrEqual', value=minFree0)]
             rv += [cls('LeadingFree1SizesGreaterOrEqual', value=minFree1)]
 
-        if "LdcEqualsLdd" not in state or state["LdcEqualsLdd"] == True:
-            rv += [cls("CDStridesEqual", value = True)]
-        else:
-            rv += [cls("CDStridesEqual", value = False)]
+        if len(state["PackedC0IndicesX"]) > 1:
```

Comment:
why use this naming CDStridesEqualInMultiFreeA?

Reply:
done